.pre-commit-config.yaml

```diff
@@ -6,19 +6,67 @@ repos:
     rev: v4.3.0
     hooks:
       - id: check-added-large-files
+        # Prevent giant files from being committed.
+      - id: check-ast
+        # Simply check whether files parse as valid python.
       - id: check-case-conflict
+        # Check for files with names that would conflict on a case-insensitive
+        # filesystem like MacOS HFS+ or Windows FAT.
+      - id: check-json
+        # Attempts to load all json files to verify syntax.
+      - id: check-merge-conflict
+        # Check for files that contain merge conflict strings.
+      - id: check-symlinks
+        # Checks for symlinks which do not point to anything.
+      - id: check-toml
+        # Attempts to load all TOML files to verify syntax.
+      - id: check-xml
+        # Attempts to load all xml files to verify syntax.
       - id: check-yaml
+        # Attempts to load all yaml files to verify syntax.
       - id: debug-statements
+        # Check for debugger imports and py37+ breakpoint() calls in python
+        # source.
+      - id: detect-private-key
+        # Checks for the existence of private keys.
       - id: end-of-file-fixer
+        # Makes sure files end in a newline and only a newline.
         exclude: ".*(data.*|extern.*|licenses.*|_parsetab.py)$"
+      - id: fix-encoding-pragma
+        # Remove # -*- coding: utf-8 -*- from the top of python files.
+        exclude: ".*(extern.*|_lextab.py|_parsetab.py)$"
```

Comment:
I think our `pyupgrade` `pre-commit` will remove these for us.  See: https://github.com/asottile/pyupgrade#-coding--comment

Reply:
Oh. good spot.

Reply:
commented out so it's not re-added at a later date.